skimage/measure/_regionprops.py

```diff
@@ -318,7 +345,7 @@ def __init__(self, slice, label, label_image, intensity_image,
         self._ndim = label_image.ndim
         self._multichannel = multichannel
         self._spatial_axes = tuple(range(self._ndim))
-        self._spacing = (spacing if spacing is not None else np.full(self._ndim, 1.))
+        self._spacing = (_check_spacing(spacing, self._ndim) if spacing is not None else np.full(self._ndim, 1.))
```

Comment:
@scikit-image/core This should probably be a library-wide utility?

Reply:
Sure. I'll move it to the core.